server/src/main/java/org/elasticsearch/ingest/IngestDocument.java

```diff
@@ -886,36 +893,6 @@ public String toString() {
         return "IngestDocument{" + " sourceAndMetadata=" + sourceAndMetadata + ", ingestMetadata=" + ingestMetadata + '}';
     }
 
-    public enum Metadata {
```

Comment:
Can we keep this here? It creates a lot of churn, and I don't think there is any real reason it needs to be inside the implementation specific map.

Reply:
Will do.